src/styles/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__rate-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 16px;
+}
+
+.card__image {
+  display: block;
+
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  width: 166px;
+  height: 36px;
+
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card__rate-reviews {
+  font-size: 10px;
+  font-weight: 400;
+
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__price-name {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-amount {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+
+  background: #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  transition: 0.3s;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
check the p.9 in the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)